cinema_service/settings.py

```diff
@@ -20,9 +21,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = (os.getenv("DJANGO_SECRET_KEY"))
```

Comment:
Security Warning: The secret key is being loaded from an environment variable. This is good practice. However, there is no default value specified. This means if the environment variable is not set, the application will crash. It's recommended to provide a default value for development environment. But remember, don't use this default value in production.